src/App.test.jsx

```diff
@@ -1,62 +1,90 @@
 // When in doubt check the docs!
 // 🚨🚨 https://mswjs.io/docs/ 🚨🚨
 
-import { screen, render } from '@testing-library/react'
+import { screen, render } from '@testing-library/react';
 // 🚨
-// import rest
-// import setupServer
-import App from './App'
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import App from './App';
 
 const user = {
   id: 1,
   created_at: '2021-12-13T00:17:29+00:00',
   // 🚨 Add a name here
-  name: '',
+  name: 'Vonta',
   avatar: 'https://thumbs.gfycat.com/NiceRequiredGrunion-size_restricted.gif',
-  header: 'https://static.wikia.nocookie.net/naruto/images/5/50/Team_Kakashi.png',
-  likes: ['React', 'Anime', 'Traveling', 'Living', 'Tower Defense Games', 'Card Games'],
+  header:
+    'https://static.wikia.nocookie.net/naruto/images/5/50/Team_Kakashi.png',
+  likes: [
+    'React',
+    'Anime',
+    'Traveling',
+    'Living',
+    'Tower Defense Games',
+    'Card Games',
+  ],
   motto: 'Res Non Verba',
   color: 'crimson',
-}
+};
 
 // 🚨 Create your server
+const server = setupServer(
+  rest.get(
+    `${process.env.REACT_APP_SUPABASE_URL}/rest/v1/users`,
+    (req, res, ctx) => res(ctx.json([user]))
+  )
+);
 
 // 🚨 Listen for server start
-beforeAll()
+beforeAll(() => server.listen());
 
 // 🚨 Close server when complete
-afterAll()
+afterAll(() => server.close);
```

Comment:
We will want to have our first mock accessible globally. We can accomplish this by placing these functions into the setupTests.js folder.